site/static/docs/5.3/assets/js/tarteaucitron.services.js

```diff
@@ -4,7 +4,7 @@ tarteaucitron.services.googletagmanager = {
   key: 'googletagmanager',
   type: 'api',
   name: 'Google Tag Manager',
-  uri: 'https://adssettings.google.com/',
+  uri: 'https://policies.google.com/privacy',
   needConsent: true,
   cookies: ['_ga', '_gat', '__utma', '__utmb', '__utmc', '__utmt', '__utmz', '__gads', '_drt_', 'FLC', 'exchange_uid', 'id', 'fc', 'rrs', 'rds', 'rv', 'uid', 'UIDR', 'UID', 'clid', 'ipinfo', 'acs'],
```

Comment:
The cookies list seems to be different as well: https://github.com/AmauriC/tarteaucitron.js/blob/fbf3a8cf27ed48b3c3d055ae65fc341d67402b1b/tarteaucitron.services.js#LL3032C5-L3033C5

Might be worth editing it.

Reply:
I think it's up to date or am I missing something ?

Reply:
Nope you're right, my comparison wasn't the right one. Resolving it.